apps/server/swagger/data-api.yml

```diff
@@ -252,3 +252,46 @@
         $ref: '#/components/responses/ServerError'
       503:
         $ref: '#/components/responses/ServiceUnavailableError'
+
+/data/category/{categoryId}/edit:
+  post:
+    summary: Edit Submitted Data. Returns an Active Submission with the records to be updated
+    tags:
+      - Admin
```

Comment:
curiosity: these tags, are they meant to imply "what user role level has access to it"? if so, what other meanings can one of these tags imply?

"admin" doesn't clearly convey what it means. a type of task, or who can do it, or what type of endpoint it is among other types?

Reply:
The idea for this tag is to facilitate the visualization of a more "risky" group of operations that would require a higher privilege like an `admin` level. 
2 operations belong in this Admin group: **update** and **delete** Submitted data. Considering that these operations could break dependencies among already Submitted Data.

Reply:
understood! great explanation!
so this is a label to group "management" tasks, that are more destructive, and therefore ideally performed by an "admin"-level user or someone else who knows what they're doing, and which we're not authenticating in terms of level permissions.

in that order of ideas, I may recommend differentiating admin as a user group, from risky tasks that should be called something else to avoid confusion.

Reply:
Just bringing more options to renaming this tag:
- "Data Management"
- "Data maintenance"
- "Data curation" 
- "Data control"
- "Critical Data Control"
Any other suggestion?

Reply:
Removed `admin` tag, as it was confusing. Left those affected endpoints under `Submission` tag as it's more representative